component_text/absrel.md

```diff
@@ -0,0 +1,97 @@
+# aBSREL
```

Comment:
Hi Stephanie, 

The directory name `component_text` is unconventional, so I wasn't sure what the intent of these markdown files were (repository documentation, text to be hosted on the site, or to be parsed and ingested by JavaScript components).